SDL/Makefile

```diff
@@ -17,13 +17,11 @@ LIB=libsdl.so
 
 CXX                  = g++
 CXXFLAGS             = -g -Wall -Wshadow -Woverloaded-virtual -fPIC -fopenmp -I..
-CXXFLAGS_CUDA        =  -g --compiler-options -Wall --compiler-options -Wshadow --compiler-options -Woverloaded-virtual --compiler-options -fPIC --compiler-options -fopenmp -dc -lineinfo --ptxas-options=-v --cudart shared -arch=compute_70 -I/mnt/data1/dsr/cub --use_fast_math --default-stream per-thread -I..
-ROOTCFLAGS           = -pthread -m64 -I/cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_11_2_0_pre5/external/slc7_amd64_gcc900/bin/../../../../../../../slc7_amd64_gcc900/lcg/root/6.20.06-ghbfee3/include
+CXXFLAGS_CUDA        = -g --compiler-options -Wall --compiler-options -Wshadow --compiler-options -Woverloaded-virtual --compiler-options -fPIC --compiler-options -fopenmp -dc -lineinfo --ptxas-options=-v --cudart shared -arch=compute_70 -I/mnt/data1/dsr/cub --use_fast_math --default-stream per-thread -I..
```

Comment:
`/mnt/data1/dsr/cub` looks like something machine specific. Is it even used?

Reply:
Seems to me like this used to be used in lnx7188 at Cornell, but now the singularity container can't see that path and the`cub` directory doesn't even exist. So I think it can be safely removed.

Reply:
can this be done in this PR, or is there a good reason to plan for another fixup PR?

Reply:
It should probably just be done here

Reply:
Removed, thank you.